src/logging_config/config.json

```diff
@@ -3,12 +3,12 @@
   "disable_existing_loggers": false,
   "formatters": {
     "simple": {
-      "format": " %(asctime)s - %(levelname)s - %(message)s",
-      "datefmt": "%Y-%m-%dT%H:%M:%S%z"
+      "format": "%(asctime)s │ %(levelname)-7s │ %(name)-17s │ %(message)s",
```

Comment:
would prefer to leave out `|` to save a bit more space, and limit `(name)` as well, i bet we can cap it at 8 chars and come up with short little nicknames for each module